src/main/data/contenttypes.json

```diff
@@ -0,0 +1,146 @@
+[
+  {
+    "code": "FTR",
+    "contentKind": {
+      "definingDoc": "SMPTE ST 429-7",
+      "scope": "http://www.smpte-ra.org/schemas/429-7/2006/CPL#standard-content",
+      "value": "feature"
+    },
+    "description": "A theatrical feature."
+  },
+  {
+    "code": "TLR",
+    "contentKind": {
```

Comment:
_contentKind_ should be _cplContentKind_

Reply:
Updated.